docs/packaging-applications/build-servers/teamcity.md

```diff
@@ -10,6 +10,10 @@ The Octopus TeamCity plugin comes with these custom build runners:
 
 1. **Octopus Deploy: Pack** Create a NuGet or Zip formatted package.
 2. **Octopus Deploy: Build Information** add information about the build, including work items and commit messages, that is then stored in Octopus Deploy.
+:::hint
+A `Branch specification` is required on the VCS configuration to populate the `Branch` field for Build Information in Octopus Deploy.  For example, `+:*`
+![](images/teamcity-branch-specification.png)
+:::
```

Comment:
Wonder if we should add a link to https://www.jetbrains.com/help/teamcity/working-with-feature-branches.html#Configuring+branches ?

"The part of the branch name matched by the asterisk (*) wildcard becomes the short branch name to be displayed in the TeamCity user-level interface (also known as the logical branch name )"

What gets sent to Octopus is the logical branch name.

Reply:
I'll add that into this PR, thanks @slewis74